### ISSUE TITLE
Question for ocsp responser hosting for Webserver 

### ISSUE THREAD

**Issue by osevan** [DorianCoding/OCSP-server#2]
Hi developer,
Is this usefull for ocsp stapling request running from localhost from nginx side? 

Or nginx got already buildin caching for this kind of stapling? 

Thanks and

Best regards

**Comment by DorianCoding** [comment]
Hi,

> Is this usefull for ocsp stapling request running from localhost from nginx side?
> 
> Or nginx got already buildin caching for this kind of stapling?

The OCSP response given by the software are indeed compatible with OCSP RFC, used by web browsers and http services such as Apache or Nginx. Those http services can cache the answer (called stapling) for the duration of the response but it is config-specific but indeed built-in. And the answer given by the software can be "stampled".

But you can also request a server-cache that will keep as well the response to avoid to do the calculations again and can work with or without an OCSP stapling from Nginx ou Apache.

In a nutshell, those "cache" are neither exclusive nor redundant. I use both in my local configuration and they work fine together.

Kind regards.
